src-ui/app/components/device-list/device-list-item/device-list-item.component.html

```diff
@@ -6,7 +6,8 @@
       </div>
       <div class="header-bar-info" (click)="editDevice()">
         <div class="device-name" translate>{{ deviceName }}</div>
-        <div class="device-serial" *ngIf="!deviceNickname">{{ deviceIdentifier }}</div>
+        <div class="device-serial" *ngIf="!deviceNickname && deviceRole" translate>{{ deviceRole }}</div>
```

Comment:
That would look like this:
```suggestion
        <div class="device-serial" *ngIf="!deviceNickname && deviceRole">{{ 'comp.device-list.deviceRole.' + deviceRole | translate }}</div>
```